src/components/ClientLibraries/ClientLibraries.styles.tsx

```diff
@@ -0,0 +1,45 @@
+import { css } from "@linaria/core";
+
+export const clientLibrary = css`
+  background-repeat: no-repeat;
+  background-position: var(--text-align-center);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 80px 0;
+  width: 100%;
+  background: #f5f7fa
```

Comment:
can we setup a theme for colors and not hardcode them here?

Reply:
we're gonna use the `@deriv/ui` colors for now. @sanjam-deriv please use the colors from `@deriv/ui`, it doesn't need to be exact match. we don't have any designs for now. just use a color close to the needed color.